code/modules/mob/mob.dm

```diff
@@ -853,4 +853,8 @@ mob/proc/yank_out_object()
 		else
 			to_chat(usr, "<span class='notice'>You just did an audible emote. Wait a while.</span>")
 			return TRUE
-	return FALSE
\ No newline at end of file
+	return FALSE
+
+// When they are affected by a queens screech
+/mob/proc/screech_act(mob/Q)
```

Comment:
```suggestion
/mob/living/proc/screech_act(mob/living/carbon/Xenomorph/Queen/Q)
```
This should be handled ideally at the `living` level, and not even consider ghosts.